backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt

```diff
@@ -278,6 +278,13 @@ class K2NativeCompilerArguments : CommonCompilerArguments() {
     )
     var preLinkCaches: String? = null
 
+    @Argument(
+            value = "-Xoverride-konan-properties",
+            valueDescription = "key1=value1;key2=value2;...",
+            description = "Override konan.properties.values"
+    )
+    var overrideKonanProperties: String? = null
```

Comment:
May be use `Array<String>?`. Argument annotation has a delimiter field, it can be set to ";". Then parsing will be easier.

Reply:
Didn't know that delimiter is adjustable. Thanks!